components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
```

Comment:
Use one language consistently. What is the difference between product and productToBuy? Maybe one of it is redundant or product could be named in the scheme of productTo...
```suggestion
export const Button = ({ setProduct, product, productToBuy }) => {
```